script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt

```diff
@@ -67,20 +67,20 @@ An outsider...?
 Then this man is not from a branch of the family?
 <Page18>
 "The Saiki family has been on an equal footing with the Tohno family since my grandfather's generation. In modern terms, we'd call them business partners.
-I'm only here on business today. It may have dragged on for a while, but I got a satisfying response. I'm heading home in a good mood.
-Aah, the last thing I want to do is spoil your homecoming with my creepy appearance. Please, excuse me."
+I'm only here on business today. It may have dragged on for a while, but I got a satisfying response. I'll be heading home in a good mood.
+Ah, spoiling your homecoming with my creepy appearance is the last thing I'd want to do. Please, excuse me."
```

Comment:
I prefer my version 
> Aah, the last thing I want to do is spoil your homecoming with my gloomy appearance. Please, excuse me."//　ああ、この陰気な姿でせっかくの帰郷に水を差すのも忍びない。今日はこのあたりで失礼するよ」

Reply:
weirdly enough, when I first read it last night, it sounded like saiki was saying he wanted to spoil his homecoming before leaving.